cpp/modmesh/view/wrap_view.cpp

```diff
@@ -487,44 +459,64 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRCameraController
-                })
-            .def(
-                "linearSpeed",
-                [](wrapped_base_type & self)
-                { return self.getLinearSpeed(); })
-            .def(
-                "lookSpeed",
-                [](wrapped_base_type & self)
-                { return self.getLookSpeed(); })
-            .def(
-                "viewCenter",
-                [](wrapped_base_type & self)
-                {
-                    const auto center = self.viewCenter();
-                    return py::make_tuple(center.x(), center.y(), center.z());
-                })
-            .def(
-                "viewVector",
+                py::arg("shift_key") = false)
+            .def_property_readonly(
+                "view_vector",
                 [](wrapped_base_type & self)
                 {
                     const auto vector = self.viewVector();
                     return py::make_tuple(vector.x(), vector.y(), vector.z());
-                })
-            .def(
-                "upVector",
-                [](wrapped_base_type & self)
-                {
-                    const auto vector = self.upVector();
-                    return py::make_tuple(vector.x(), vector.y(), vector.z());
                 });
+
+#define DECL_QVECTOR3D_PROPERTY(NAME, GETTER, SETTER)          \
+    .def_property(                                             \
+        #NAME,                                                 \
+        [](wrapped_type & self)                                \
+        {                                                      \
+            QVector3D const v = self.GETTER();                 \
+            return py::make_tuple(v.x(), v.y(), v.z());        \
+        },                                                     \
+        [](wrapped_type & self, std::vector<double> const & v) \
+        {                                                      \
+            double const x = v.at(0);                          \
+            double const y = v.at(1);                          \
+            double const z = v.at(2);                          \
+            self.SETTER(QVector3D(x, y, z));                   \
+        })
+
+        (*this)
+            // clang-format off
+                    DECL_QVECTOR3D_PROPERTY(position, position, setPosition)
+                    DECL_QVECTOR3D_PROPERTY(up_vector, upVector, setUpVector)
+                    DECL_QVECTOR3D_PROPERTY(view_center, viewCenter, setViewCenter)
+                    DECL_QVECTOR3D_PROPERTY(default_position, defaultPosition, setDefaultPosition)
+                    DECL_QVECTOR3D_PROPERTY(default_view_center, defaultViewCenter, setDefaultViewCenter)
+                    DECL_QVECTOR3D_PROPERTY(default_up_vector, defaultUpVector, setDefaultUpVector)
+            // clang-format on
+            ;
+#undef DECL_QVECTOR3D_PROPERTY
+
+#define DECL_FLOAT_PROPERTY(NAME, GETTER, SETTER) \
```

Comment:
The macro naming doesn't follow the style rules.
https://github.com/solvcon/modmesh/blob/master/STYLE.rst#c-pre-processor-macro